apps/token-manager/app/src/components/TokenBadge.js

```diff
@@ -8,7 +8,7 @@ class TokenBadge extends React.PureComponent {
       <Main title={`${name} (${address})`}>
         <Label>
           <Icon src={`https://chasing-coins.com/coin/logo/${symbol}`} />
-          {name === symbol ? name : `${name} (${symbol})`}
+          <Name>{name === symbol ? name : `${name} (${symbol})`}</Name>
```

Comment:
I'm wondering if we should capture the symbol first, and then leave the name in the parentheses. They can both be arbitrarily long strings, but I think it's more useful to have the short representation by default (in case the name cuts off the symbol). They're both as easily spoofable, so the full badge would have to be inspected if you wanted to be absolutely 100% sure.

Reply:
Good idea 👍 

<img width="317" alt="image" src="https://user-images.githubusercontent.com/36158/47643131-cf8de780-db6a-11e8-82c2-fa0b650bfac9.png">